static/scss/general_styles.scss

```diff
@@ -0,0 +1,45 @@
+// General Style Rules
+
+body {
+  background-color: $page-background;
+  font-family: 'Roboto', helvetica, arial, sans-serif !important;
+  line-height: 1.4em;
+  }
+
+
+ .mdl-card {
+    padding: 20px 24px 30px;
+    margin: 0 0 22px 0;
+    min-height: 150px;
+    width: 100%;
+    border-radius: 1px;
+
+	.mdl-card__title {
+	padding: 0;
+	margin: 0 0 20px;
+
+
+	}
+
+    .mdl-card__title-text {
+      font-weight: 500;
+      font-size: 13pt;
+      font-weight: 500;
+      color: $font-black;
+    }
+  }
+
+
+.mdl-shadow--2dp, .mdl-shadow--3dp {box-shadow: none;}
```

Comment:
Can you put each CSS directive on its own line? Ie

    .mdl-shadow--2dp, .mdl-shadow--3dp {
      box-shadow: none;
    }